### DISCUSSION TITLE
How to add cc email on button Email List of All Assigned

### DISCUSSION THREAD

**Discussion by yprasetiyo** [snipe/snipe-it#12540] [General]
Hi All,
Can you help me how to add cc email in the email list of all assigned button in the user menu.

![image](https://user-images.githubusercontent.com/66456227/220241431-b1f57bca-d6e5-4581-801c-ae1a88c42220.png)

**Comment by gregory-packet** [comment]
+1

Is there also a way to edit the text and email referenced in the email? Our IT manager's email is referenced in the email, but he isn't the person who manages assets in this particular lab. I'd like to spare him from his inbox being flooded. 

A way of bulk sending would be incredible as well.
